lib/src/coord_each.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:turf/helpers.dart';
+
+coordEach(GeoJSONObject? geoJson, Function(Position, int coordIndex, int featureIndex, int multiFeatureINdex, int geometryIndex) callback,
```

Comment:
there is already another approach to implement this at #50

Reply:
Let's see if we can maybe use this implementation instead, because it's already very well tested.

CC @tobrun

Reply:
No problem. I'll have a look.

Reply:
thanks for ping @lukas-h and awesome for contributing @Dennis-Mwea . I was planning to get back to #50 today.
The idea is that we align the coordEach implementation with geomEach setup from @baparham (pending doing some benchmarks). The micro optimizations done on turf-js might not be needed with dart (to be validated).

Reply:
Changed to the existing implementation and removed the duplicate